components/comments/CommentBubble.tsx

```diff
@@ -47,22 +84,38 @@ const CommentBubble: React.FC<ChatBubbleProps> = ({ firstName, lastName, timesta
       </div>
       {menuOpen && (
         <div className="absolute right-0 mt-2 py-1 w-48 bg-white rounded-md shadow-lg z-50">
-          <ul>
-            <li className="flex items-center px-4 py-2 text-sm text-gray-700 hover:bg-gray-100">
-              <PencilSquareIcon className="w-5 h-5 mr-2" />
-              Edit
-            </li>
-            <li className="flex items-center px-4 py-2 text-sm text-gray-700 hover:bg-gray-100">
-              <TrashIcon className="w-5 h-5 mr-2" />
-              Delete
-            </li>
-          </ul>
+          <button
+            onClick={() => {} /* handleEdit */}
+            className="flex items-center px-4 py-2 text-sm text-gray-700 hover:bg-gray-100 w-full text-left"
+          >
+            <PencilSquareIcon className="w-5 h-5 mr-2" />
+            Edit
+          </button>
+          <button
+            onClick={handleDelete}
+            className="flex items-center px-4 py-2 text-sm text-gray-700 hover:bg-gray-100 w-full text-left"
+            aria-label="Delete comment"
+          >
+            <TrashIcon className="w-5 h-5 mr-2" />
+            Delete
+          </button>
         </div>
       )}
       <div className={bodyStyles}>
         <p>{text}</p>
       </div>
       <div className={isAuthor ? userTailStyles : otherUserTailStyles}></div>
+      {showConfirm && (
```

Comment:
as the component has `isOpen` prop, can you check if this condition is not neccessary?